notices.txt

```diff
@@ -0,0 +1,89 @@
+=========================================================================
+==  NOTICE file corresponding to section 4 d of            	 	  ==
+==  the Apache License, Version 2.0,                                   ==
+==  in this case for NativeScript Doctor v0.9					  ==
+=========================================================================
+
+NativeScript Doctor v0.9 (the �Product�)

```

Comment:
https://github.com/NativeScript/nativescript-doctor/blob/master/package.json#L3 the current version is 0.12.0

Reply:
The problem is we have not had this file before. I had to create one for Sidekick. But now we have the Sidekick's notice.txt and you are right I that we could request new one.